### PR TITLE
Remove flux store usages from websocket_actions.jsx

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Remove flux store usages from websocket_actions.jsx.

@lindy65, could you regression test the following logged in on multiple browsers: leaving a team, view an unread channel on the same team, updating a user, deleting a team, getting removed from a channel you're viewing.

#### Ticket Link
Part of https://mattermost.atlassian.net/browse/MM-12564

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

**Comment by jwilander** [comment]
Dev reviewers, might be easier to look at commit by commit

File: actions/websocket_actions.jsx [hunk 1]
```diff
@@ -739,10 +742,10 @@ function handleReactionRemovedEvent(msg) {
 
 function handleChannelViewedEvent(msg) {
     // Useful for when multiple devices have the app open to different channels
-    if ((!window.isActive || ChannelStore.getCurrentId() !== msg.data.channel_id) &&
+    if ((!window.isActive || getCurrentChannelId(getState()) !== msg.data.channel_id) &&
         UserStore.getCurrentId() === msg.broadcast.user_id) {
-        // Mark previous and next channel as read
-        ChannelStore.resetCounts([msg.data.channel_id]);
+        console.log('yo');
```
Comment (saturninoabril) [inline actions/websocket_actions.jsx]: Maybe forgot to remove?

Comment (jwilander) [reply actions/websocket_actions.jsx]: Removed in a later commit :)

**Comment by lindy65** [comment]
@jwilander can you set up a new spinmint please? I'm getting a weird redirecting loop and login page is not loading. (For future, how do I destroy an existing spinmint and create a new one?)

**Comment by jwilander** [comment]
@lindy65 if you remove the label, delete all the mattermod comments and then add the label back it will generate a new spinmint for you. There seems to be some sort of issue with mattermod at the moment, however

**Comment by lindy65** [comment]
Thanks @jwilander I'll try again in a while

**Comment by lindy65** [comment]
Thanks @jwilander 

I tested being logged in on multiple browsers: 
1. leaving a team, 
    - in the browser window I leave the team from, I get the following error message and am not directly taken to the remaining team I belong to.  After clicking on "Back to Town Square", I am taken to Town Square of the remaining team I belong to. (In the other browser window I am observing from, I am taken to Town Square of the remaining team I am on (as expected) without the attached error message.)

image

   - This exact scenario does NOT reproduce on `master` but there is a bug on `master` where when you leave a team, the team icon in the sidebar disappears along with public and private channels but the team name is still at the top of the LHS next to your username

image

   - I'm unsure whether the issue on `master` is causing the issue on this particular PR or whether they're unrelated?  Let me know and I can put in a separate ticket for the issue on `master`.

2. view an unread channel on the same team, 
    - no issues found

3. updating a user, 
    - no issues found

4. deleting a team, 
    - I'm not able to test this yet as I believe this is done via CLI?

5. getting removed from a channel you're viewing.
    - no issues found (message advising I've been removed from the channel is displayed as expected).

**Comment by jwilander** [comment]
Thanks @lindy65, I'll check 1)

**Comment by jwilander** [comment]
@lindy65 does 1) reproduce for you every time? I can't seem to reproduce it locally or on the spinmint even when throttling my connection

**Comment by lindy65** [comment]
Hey @jwilander - I've just retested and, as sods law would have it, I am no longer able to reproduce :(  Tried both sysadmin and user-1 accounts, Chrome, Firefox and IE11 and couldn't get the issue to reproduce again...

**Pull request merged**